### PR TITLE
Add library dirs to luajit target

### PR DESCRIPTION
**Pull request by cmcfarlen**

### CODE REVIEW

**Review by zwoop — APPROVED** [review]

File: cmake/FindLuaJIT.cmake [hunk 1]
```diff
@@ -44,5 +44,6 @@ endif()
 if(LuaJIT_FOUND AND NOT TARGET LuaJIT::LuaJIT)
     add_library(LuaJIT::LuaJIT INTERFACE IMPORTED)
     target_include_directories(LuaJIT::LuaJIT INTERFACE ${LuaJIT_INCLUDE_DIRS})
+    target_link_directories(LuaJIT::LuaJIT INTERFACE ${LuaJIT_LIBRARY_DIRS})
     target_link_libraries(LuaJIT::LuaJIT INTERFACE ${LuaJIT_LIBRARIES})
```
Comment (JosiahWI) [inline cmake/FindLuaJIT.cmake:48]: Is this line still needed?

Comment (cmcfarlen) [reply cmake/FindLuaJIT.cmake:48]: `LuaJIT_LIBRARIES` just has the `luajit-5.1`, but if that isn't in a known path then it fails to find it.  So I think it needs both.

Comment (cmcfarlen) [reply cmake/FindLuaJIT.cmake:48]: From CMakeCache.txt

```
LuaJIT_LIBRARY_DIRS:INTERNAL=/opt/homebrew/Cellar/luajit/2.1.0-beta3-20230430.1/lib
LuaJIT_LIBRARIES:INTERNAL=luajit-5.1
```

Comment (JosiahWI) [reply cmake/FindLuaJIT.cmake:48]: Interesting. Thanks for the fix.

**Review by JosiahWI — APPROVED** [review]

**Review by bneradt — APPROVED** [review]

**Pull request merged**